test/otp/utils.go

```
@@ -0,0 +1,693 @@
+	if strings.Count(svcNetwork, ":") >= 2 && strings.Count(svcNetwork, ".") >= 2 {
+		return "dualstack"
+	} else if strings.Count(svcNetwork, ":") >= 2 {
+		return "ipv6single"
+	} else if strings.Count(svcNetwork, ".") >= 2 {
+		return "ipv4single"
+	}
+	return ""
+}
+
+func nslookDomainName(domainName string) string {
+	ips, err := net.LookupIP(domainName)
```

golangci-lint (2.12.2)

[error] net.LookupIP must not be called. use (*net.Resolver).LookupIPAddr with a context

(noctx)